src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const discountOfSevenDays = 50;
+  const discountOfThreeOrMoreDays = 20;
+  const total = days * 40;
```

Comment:
Consider extracting the `40` into a constant with a descriptive name such as `DAILY_RATE`, to avoid magic numbers and make the code more readable.